core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphqlActionBuilder.kt

```diff
@@ -0,0 +1,127 @@
+package org.evomaster.core.problem.graphql
+
+import graphql.language.*
+import graphql.schema.idl.TypeDefinitionRegistry
+import org.evomaster.core.problem.graphql.param.GraphqlParam
+import org.evomaster.core.problem.rest.param.Param
+import org.evomaster.core.search.Action
+import org.evomaster.core.search.gene.*
+import org.slf4j.LoggerFactory
+import org.slf4j.Logger
+import java.lang.IllegalStateException
+import java.util.concurrent.atomic.AtomicInteger
+
+
+class GraphqlActionBuilder {
+
+    companion object {
+        private val log: Logger = LoggerFactory.getLogger(GraphqlActionBuilder::class.java)
+        private val idGenerator = AtomicInteger()
+
+        fun addActionsFromSchema(schema: TypeDefinitionRegistry,
+                                 actionCluster: MutableMap<String, Action>) {
+
+            actionCluster.clear()
+
+            val rootOperations = mutableListOf<String>("Mutation", "Query")
+
+            rootOperations.forEach{
+                val obj = schema.getType(it).get() as ObjectTypeDefinition
+                val operationType = it
```

Comment:
instead of creating `val operationType = it`, you could directly use something like `.forEach{ operationType -> `